packages/common/src/system/guid-utils.ts

```diff
@@ -8,21 +11,22 @@ import * as uuid from 'uuid-with-v6';
 /**
  * UUID RFC 4122 https://tools.ietf.org/html/rfc4122
  */
-export namespace Guid {
+@injectable()
+export class GuidUtils {
```

Comment:
why not call it like guid-generator or provider or smth? Utils implies smth generic like a kitchen sink